tests/net_dynamic_hb.rs

```diff
@@ -278,22 +272,21 @@ fn do_drop_and_readd(cfg: TestConfig) {
             }
         }
 
-        let (era, hb_epoch) = state.get_net_mut()[node_id].algorithm().algo().epoch();
+        let (era, hb_epoch) = state.net[node_id].algorithm().algo().epoch();
         if node_id != pivot_node_id
             && awaiting_addition_input.contains(&node_id)
             && state.shutdown_epoch.is_some()
             && era + hb_epoch == state.shutdown_epoch.unwrap()
         {
             // Now we can add the node again. Public keys will be reused.
-            let step = state
-                .get_net_mut()
+            let _ = state
```

Comment:
This is still a `Step`, isn't it? Let's `assert!` that its output is empty instead of dropping it right away.
(We don't actually expect the step to be completely empty, because it will contain a `SignedVote` message, right?)